src/curve/edwards/extended.rs

```diff
@@ -414,31 +414,21 @@ impl<'a, 'b> Mul<&'b ExtendedPoint> for &'a Scalar {
 
 #[cfg(test)]
 mod tests {
+    use hex_literal::hex;
     use super::*;
-    use hex::decode as hex_decode;
-    use std::convert::TryInto;
-    fn slice_to_fixed_array(b: &[u8]) -> [u8; 56] {
-        let mut a: [u8; 56] = [0; 56];
-        a.copy_from_slice(&b);
-        a
-    }
-
-    fn hex_to_field(data: &str) -> FieldElement {
```

Comment:
Would prefer to keep these are functions instead of introducing `macro_rules!` also I'd prefer `hex_to_field` instead of `hex2field`

Reply:
`hex_literal::hex!` requires a string **literal**.
But we can use the **internal** function `hex_literal::decode` so we can receive a `&'static str`.